app/Message.php

```diff
@@ -35,14 +35,14 @@ class Message extends Model
     public $timestamps = true;
 
     // Messages status, they all start at 0
-    const POSTED = 100; // Message has been successfuly converted to discussion or wathever
-    const NEEDS_VALIDATION = 10; // message needs to be validated by poster or admin
-    const CREATED = 0; // message has just been imported from the mail server (default)
-    const BOUNCED = -10; // message bounced back to user
-    const INVALID = -20; // message cannot be posted to a group (group not found...)
-    const AUTOMATED = -30; // message is an autoreply or away message
-    const ERROR = -50; // message cound not be converted to content
-    const SPAM = -100; // message is spam
+    protected const POSTED = 100; // Message has been successfully converted to discussion or whatever
```

Comment:
The fact that it is marked as protected probably means this class is not used anywhere anymore. I will investigate.